polaris-react/src/components/Frame/components/Toast/Toast.tsx

```diff
@@ -53,21 +56,37 @@ export function Toast({
   );
 
   const actionMarkup = action ? (
-    <div className={styles.Action}>
+    <Box marginLeft="8" marginRight="4">
       <Button plain monochrome onClick={action.onAction}>
         {action.content}
       </Button>
-    </div>
+    </Box>
   ) : null;
 
-  const className = classNames(styles.Toast, error && styles.error);
+  const breakpoints = useBreakpoints();
+  const padding = breakpoints.mdUp ? '4' : '2';
+  const textVariant = breakpoints.mdUp ? 'headingLg' : 'headingMd';
 
   return (
-    <div className={className}>
-      <KeypressListener keyCode={Key.Escape} handler={onDismiss} />
-      {content}
-      {actionMarkup}
-      {dismissMarkup}
-    </div>
+    <Box
+      borderRadius="2"
+      background={error ? 'action-critical' : 'surface-dark'}
+      color={error ? 'text-on-critical' : 'text-on-dark'}
+      marginBottom="5"
+      maxWidth="500"
```

Comment:
This could be done on the parent right? Wondering if it's worthwhile to add max width to box.

Reply:
I'm going to drop any changes to the `Toast` component before merging this into our prototype branch so that our alpha release isn't altering any existing Polaris components (that would be in the beta release). But a majority of the time, I see `Toast` being used inside `Frame` and I wasn't sure if `Frame` should be responsible for max width, which is why I decided to put it in box. I can ship this with max width support on `Box` for now and we can iterate in the beta phase 👍